src/go-contactless.html

```diff
@@ -130,20 +118,60 @@ <h3>Transport for London</h3>
       </picture>
       <h3>MTA New York City Transit</h3>
       <p>
-        On the first day that MTA launched contactless fares—at just 16 subway stations—nearly 6,000 riders tapped to
-        pay: 80% with a smart device and 20% with a bank card.
+        On the first day that MTA launched contactless fares—at just 16 subway stations—nearly 6,000 riders tapped to pay: 80%
+        with a smart device and 20% with a bank card.
       </p>
     </div>
   </div>
 </section>
 
-<hr>
-
-<section class="go-contactless-section pb-0 text-center" id="call-to-action">
+<section class="go-contactless-section go-contactless-timeline-container bg-dark-tan" id="call-to-action">
   <header>
-    <h2 class="mt-0">Ready to tap into the future of fare payment?</h2>
+    <h2 class="mt-0 mb-4 pb-3 text-center">Ready to tap into the future of fare payment?</h2>
   </header>
-  <section>
-    <a class="btn btn-outline-dark" href="/contracts">Get Started</a>
-  </section>
+  <div class="row mt-4 pt-3 d-flex flex-column-reverse flex-md-row">
+    <div class="procurement-image col-md-4 col-lg-4 text-center">
+      <img
+        alt="A smart phone and credit card being tapped at a bus fare device"
+        class="img-fluid contactless-cta-image"
+        src="/uploads/go-contactless-illo-payment-types.png"
```

Comment:
I'm curious about the reasoning to have separate files for home vs. go-contactless

Reply:
The backgrounds are different colors: https://github.com/cal-itp/mobility-marketplace/pull/386#issuecomment-1301417829

Reply:
Oh oops, I totally missed that there was some earlier discussion in this PR! Thanks